dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/response/SolrSearchResult.java

```diff
@@ -107,13 +103,11 @@
     private boolean inReviewState = false;
     private boolean deaccessionedState = false;
     private Long datasetVersionId;
-    private String versionNumberFriendly;
     //Determine if the search result is owned by any of the dvs in the tree of the DV displayed
     private boolean isInTree;
     private float score;
     private List<String> userRole;
     private boolean harvested = false;
-    private String dvTree;
     private List<String> fileCategories = null;
     private List<String> tabularDataTags = null;
```

Comment:
From what I could tell:
`private List<String> userRole;` - is always null
`private DatasetThumbnail datasetThumbnail;` - is not used